app/validators/claim/advocate_interim_claim_validator.rb

```diff
@@ -30,4 +26,14 @@ def validate_earliest_representation_order
     return unless date.present?
     add_error(:base, 'unclaimable') unless date >= Date.parse(Settings.agfs_fee_reform_release_date.to_s)
   end
+
+  def validate_offence
+    validate_presence(:offence, 'blank')
```

Comment:
Also not tested

Reply:
This one will get addressed when I come to do the work on the offences themselves ;)